template_scripts/fw_creel.Rmd

```diff
@@ -834,4 +846,43 @@ if(nchar(params$output_location_filepath) > 1) {
 #   path = file.path(where("fishery_analyses", params$project_name, params$fishery_name), paste0("bss_overview_", params$fishery_name, ".xlsx"))
 #   )
 # }
+
+#Export data from one or both models
+if (params$export %in% c("database", "local") && !params$model_used %in% c("PE only", "BSS only", "Both models" )) {
+  cat("Invalid value for model_used parameter. Must select 'PE only', 'BSS only', or 'Both models'.")
+} else {
+
+  # Initialize objects for storing processed data
+  # also serves as placeholder for other model if only one was used
+  transformed_pe_data <- list(
+    pe_effort = data.frame(),
+    pe_catch = data.frame(),
+    pe_stratum_effort = data.frame(),
+    pe_stratum_catch = data.frame(),
+    pe_summarized_effort = data.frame(),
+    pe_summarized_catch = data.frame()
+  )
+  
+  transformed_bss_data <- list(
+    bss_stratum = data.frame(),
+    bss_summarized = data.frame(),
+    bss_stratum_catch = data.frame(),
+    bss_stratum_effort = data.frame(),
+    bss_summarized_catch = data.frame(),
+    bss_summarized_effort = data.frame()
+  )
+  
+  if (params$model_used == "PE only") {
+    export_estimates(params, estimates_pe, estimates_bss = NULL)
+  }
+  
+  if (params$model_used == "BSS only") {
+    export_estimates(params, estimates_pe = NULL, estimates_bss)
+  }
+  
+  if (params$model_used == "Both models") {
+    export_estimates(params, estimates_pe, estimates_bss)
+  }
+}
+
 ```
```

Comment:
Nvm, kept getting this warning. Might need a hand hold on setting up the yml config file 😄

Reply:
I will read more into the yaml package documentation and develop standard set of instructions for setting up config file, database connection, etc. to avoid this warning.
https://github.com/vubiostat/r-yaml